SQL/New_patches/2019_07_05_Add_Language_feature.sql

```diff
@@ -1,12 +1,14 @@
-CREATE TABLE `test_names_fr` (
+CREATE TABLE `test_names_dictionary` (
   `ID` int(11) NOT NULL AUTO_INCREMENT,
   `Test_name` varchar(255) DEFAULT NULL,
-  `Full_name_fr` varchar(255) NOT NULL,
+  `Full_name_alternative` varchar(255) NOT NULL,
+  `language_code` varchar(255) NOT NULL,
```

Comment:
I think this should be a foreign key to a languages table, not a varchar, so that there's still an enumeration of supported languages by the loris instance (especially for things like dropdowns where you need to select the language.)

Reply:
it is a foreign key to the language table.

Reply:
oops, missed that..